### DISCUSSION TITLE
Why is the highlight so bright in nvim tree

### DISCUSSION THREAD

**Discussion by rew1nter** [nvim-tree/nvim-tree.lua#2802] [Q&A] [answered]
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/64508791/e36b800f-6140-40cc-b7a3-9ef3e3c81cfe)
Is it possible to make the highlight less bright to equal the editor highlight intensity

**Comment by alex-courtis** [comment, marked as answer]
You can change all highlighting and style of nvim-tree, see `:help nvim-tree-highlight`
